Adding my kennard-stone package

yu9824 opened the pull request:
<!--
Thank you very much for putting in this recipe PR!

This repository is very active, so if you need help with a PR, please let the
right people know. There are language-specific teams for reviewing recipes.

| Language        | Name of review team           |
| --------------- | ----------------------------- |
| python          | `@conda-forge/help-python`    |
| python/c hybrid | `@conda-forge/help-python-c`  |
| r               | `@conda-forge/help-r`         |
| java            | `@conda-forge/help-java`      |
| nodejs          | `@conda-forge/help-nodejs`    |
| c/c++           | `@conda-forge/help-c-cpp`     |
| perl            | `@conda-forge/help-perl`      |
| Julia           | `@conda-forge/help-julia`     |
| ruby            | `@conda-forge/help-ruby`      |
| other           | `@conda-forge/staged-recipes` |

Once the PR is ready for review, please mention one of the teams above in a
new comment. i.e. `@conda-forge/help-some-language, ready for review!`
Then, a bot will label the PR as 'review-requested'.

Due to GitHub limitations, first time contributors to conda-forge are unable
to ping conda-forge teams directly, but you can [ask a bot to ping the team][1]
using a special command in a comment on the PR to get the attention of the
`staged-recipes` team. You can also consider asking on our [Gitter channel][2]
if your recipe isn't reviewed promptly.

[1]: https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-ping-team
[2]: https://gitter.im/conda-forge/conda-forge.github.io
-->

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipes/kennard-stone```) and found some lint.

Here's what I've got...


For **recipes/kennard-stone**:

 * license_file entry is missing, but is required.
 * noarch: python recipes are required to have a lower bound on the python version. Typically this means putting `python >=3.6` in **both** `host` and `run` but you should check upstream for the package's Python compatibility.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/kennard-stone```) and found it was in an excellent condition.

chrisburr commented on recipes/kennard-stone/meta.yaml:
```diff
@@ -0,0 +1,45 @@
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/7a/ce/a4e3b09323ed9e7ae0054795927ba62d1d9bfc5d3e09f0d92b8706a50e06/kennard_stone-1.1.1.tar.gz"
```
This should use the predictable form of the URL templated with `{{ version }}`. See `recipes/example/meta.yaml` for an example.

chrisburr commented on recipes/kennard-stone/meta.yaml:
```diff
@@ -0,0 +1,45 @@
+
+requirements:
+  host:
+    - numpy
+    - pandas
+    - pip
+    - python >=3.6
+    - scikit-learn
```
Nominally pure Python packages should only need `python` and `pip` in `host`:
```suggestion
    - pip
    - python >=3.6
```

chrisburr commented on recipes/kennard-stone/meta.yaml:
```diff
@@ -0,0 +1,45 @@
+  license_family: MIT
+  license_file: LICENSE 
+  summary: "A method for selecting samples by spreading the training data evenly."
+  doc_url: 
+  dev_url: 
```
```suggestion
```

yu9824 commented:
Thank you for your review and comments.

I have modified the `meta.yaml` based on your point. Specifically, here is what we have done
- Changed the `url` to one that uses variables.
- Removed `doc_url:` and `dev_url:` entries.
- Changed `host:` to only pip and python.
